src/main/java/newbank/server/NewBankClientHandler.java

```diff
@@ -75,7 +75,10 @@ public void run() {
     try {
       while (hasMore) {
         String request = in.readLine();
-        out.println(String.format("Received request [%s]", request));
+        if (NewBankServer.VERBOSE_MODE) {
+          // useful for debugging/testing, not meant for production
+          out.println(String.format("Received request [%s]", request));
```

Comment:
Aren't we relying on those in tests?

Reply:
Yes. The tests rely on those so it hasn't changed.